next.config.js

```diff
@@ -10,6 +13,16 @@ const nextConfig = {
       },
     ],
   },
+  webpack: (config) => {
+    // Ensure transpiling of the correct entry file
+    config.resolve.alias['slate-transcript-editor'] = require.resolve('slate-transcript-editor/dist');
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      react: path.resolve('./node_modules/react'),
```

Comment:
This was needed while I was linking a local instance of the `slate-transcript-editor` for development purposes as it was picking the versions of react in that one. Just checked and those are not needed with the current setup which consumes the package from the GitHub repo. I assume that this will also be the case if/when we use a published package.

Good catch! I will remove it, and check if it still needed when developing locally.

Reply:
This is still needed when developing with a local slate-transcript-editor. Removing it would create issues for anyone trying to do that in the future, so I decided to leave it and add a comment there to explain why the code is there.

There might be a way to make those aliases obsolete, but I already spend too much time trying to do that, so I leave it as it is for now.

Reply:
sounds good